testbed/tests/brakelight_test.py

```diff
@@ -0,0 +1,43 @@
+# firmware/testbed/tests/brakelight_test.py
+
+import time
+from testbeds.hil_testbed import teststand
+from drivers.common_drivers.can_driver import HILBoard, VehicleBoard
+from tqdm import tqdm
+
+def test_steeringAngle(teststand):
+    print(teststand.vehicle_boards)
+    vcu_hil: HILBoard = teststand.hil_boards["vcu_hil"]
+    # pdu_hil: HILBoard = teststand.hil_boards["pdu_hil"]
```

Comment:
should remove unused code to keep it clean

Reply:
I don't think reading from the PDU is necessary if the goal is to just determine if the PDU enables the brake light. Should read from the PDU HIL since it relays the PDU outputs and remove the PDU.

Reply:
@DanBishara, this is temporary because I think the wire harness for the PDU HIL is not done so Aryan told them to read from the car's ECUs. Regarding this, @aryanGajelli @Justin-Vuong I don't think we should be merging this until this has been tested with the PDU HIL.

Reply:
Ah okay I see, do you know what the current updates on that are?